cylc/flow/simulation.py

```diff
@@ -76,13 +102,14 @@ def configure_sim_modes(taskdefs, sim_mode):
         )
 
 
-def get_simulated_run_len(rtc: Dict[str, Any]) -> int:
-    """Get simulated run time.
+def set_simulated_run_len(rtc: Dict[str, Any]) -> int:
```

Comment:
I would change this back to `get_simulated_run_len` as there is no longer any change to the code